### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by azakus**
Remove testing polyfilled Custom Elements with native ShadowDOM.
Fix Chrome tests by using the `sudo: required` Travis Configuration due to travis-ci/travis-ci#8836

### CODE REVIEW

**Comment by azakus** [comment]
Safari is just taking a long time to run tests. Need to investigate reducing overhead in testing.

File: test/runner.html [hunk 1]
```diff
@@ -81,51 +82,39 @@
...
       return previous + (previous ? '&' : '') + next;
     }
 
     // test shadowdom/custom elements polyfills together
     // preferring both if possible.
-    var matrix = [''];
+    let flags = [''];
     if (window.customElements) {
-      matrix.push('wc-ce=true');
+      // flags.push('wc-ce=true');
```
Comment (sorvell) [inline test/runner.html]: Let's remove or explain why we're not testing this.

File: test/runner.html [hunk 2]
```diff
@@ -81,51 +82,39 @@
...
     }
 
     // economize testing by testing css shimming
     // only against 1 environment (native or polyfill).
-    if (window.CSS && CSS.supports && CSS.supports('box-shadow', '0 0 0 var(--foo)')) {
-      var last = matrix.length === 1 ? '' : matrix[matrix.length - 1];
-      matrix.push(addUrlOption(last, 'wc-shimcssproperties=true'));
-    }
-    suites = combinations(suites, [matrix]);
+    // if (window.CSS && CSS.supports && CSS.supports('box-shadow', '0 0 0 var(--foo)')) {
```
Comment (sorvell) [inline test/runner.html]: Same point here.

File: util/travis-sauce-test.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+set -x
+wct -s 'windows 10/microsoftedge@14' -s 'windows 10/microsoftedge@15' -s 'windows 8.1/internet explorer@11' -s 'os x 10.11/safari@9' -s 'macos 10.12/safari@10' -s 'macos 10.13/safari@11' -s 'Linux/chrome@41'
```
Comment (sorvell) [inline util/travis-sauce-test.sh]: Let's try adding Edge 16

Comment (azakus) [reply util/travis-sauce-test.sh]: Done!

File: test/runner.html (hunk 2 above)
Comment (azakus) [reply test/runner.html]: Done

File: test/runner.html (hunk 1 above)
Comment (azakus) [reply test/runner.html]: Done

**Comment by azakus** [comment]
Edge 16 tests failing due to https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12084341/

**Comment by azakus** [comment]
Removing Edge 16 testing for now to get master back to "green", then we can readd.

**Pull request merged**